src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -161,11 +161,21 @@ protected override void StopProcessing()
                     FileName = "/sbin/shutdown",
                     Arguments = string.Empty,
                     RedirectStandardOutput = false,
+                    RedirectStandardError = true,
                     UseShellExecute = false,
                     CreateNoWindow = true,
                 }
             };
             _process.Start();
+            _process.WaitForExit();
+            if (_process.ExitCode != 0)
+            {
+                string stderr = _process.StandardError.ReadToEnd();
+                string errMsg = StringUtil.Format("Command returned 0x{0:X}: {1}", _process.ExitCode, stderr);
+                ErrorRecord error = new ErrorRecord(
+                    new InvalidOperationException(errMsg), "CommandFailed", ErrorCategory.OperationStopped, null);
```

Comment:
"CommandFailed" exists in ConsoleHostStrings.resx but it seems it is not used.

Reply:
Thanks, will look into that.  Still trying to figure out why the cmdlet fails differently in a container.

Reply:
Removed